icons/picture-in-picture.svg

```diff
@@ -9,6 +9,9 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M8 4.5v5H3m-1-6 6 6m13 0v-3c0-1.16-.84-2-2-2h-7m-9 9v2c0 1.05.95 2 2 2h3" />
-  <rect width="10" height="7" x="12" y="13.5" ry="2" />
+  <path d="M21 9V6a2 2 0 0 0-2-2h-7" />
+  <path d="M3 13v2a2 2 0 0 0 2 2h3" />
+  <path d="m3 4 5 5" />
+  <path d="M3 9h5V4" />
+  <rect x="12" y="13" width="10" height="7" rx="2" />
```

Comment:
This feels off centre, how about something like this?

This arrow is also more consistent in size with arrows in calendar/clock/square/circle-up/down

![image](https://github.com/user-attachments/assets/0734e51a-f968-4bb9-9ecd-5f04dd6c2a14)
![image](https://github.com/user-attachments/assets/3cce68a6-c732-47fe-9e27-6fe9db23fabd)

<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M2+10h6V4%22+/%3E%3Cpath+d=%22m2+4+6+6%22+/%3E%3Cpath+d=%22M21+10V7a2+2+0+0+0-2-2h-7%22+/%3E%3Cpath+d=%22M3+14v2a2+2+0+0+0+2+2h3%22+/%3E%3Crect+x=%2212%22+y=%2214%22+width=%2210%22+height=%227%22+rx=%221%22+/%3E&name=icon&base=icon"><img alt="icons" width="400px" src="https://lucide.dev/api/gh-icon/PHN2ZwogIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICB3aWR0aD0iMjQiCiAgaGVpZ2h0PSIyNCIKICB2aWV3Qm94PSIwIDAgMjQgMjQiCiAgZmlsbD0ibm9uZSIKICBzdHJva2U9ImN1cnJlbnRDb2xvciIKICBzdHJva2Utd2lkdGg9IjIiCiAgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIgogIHN0cm9rZS1saW5lam9pbj0icm91bmQiCj4KICA8cGF0aCBkPSJNMiAxMGg2VjQiIC8+CiAgPHBhdGggZD0ibTIgNCA2IDYiIC8+CiAgPHBhdGggZD0iTTIxIDEwVjdhMiAyIDAgMCAwLTItMmgtNyIgLz4KICA8cGF0aCBkPSJNMyAxNHYyYTIgMiAwIDAgMCAyIDJoMyIgLz4KICA8cmVjdCB4PSIxMiIgeT0iMTQiIHdpZHRoPSIxMCIgaGVpZ2h0PSI3IiByeD0iMSIgLz4KPC9zdmc+.svg"/><br/>Open lucide studio</a>